app/src/main/res/layout/fragment_suggested_edits_add_descriptions.xml

```diff
@@ -80,44 +80,79 @@
         <androidx.constraintlayout.widget.ConstraintLayout
             android:id="@+id/bottomButtonContainer"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:paddingBottom="8dp"
+            android:layout_height="50dp"
             android:clipToPadding="false"
+            android:paddingBottom="8dp"
+            android:layout_marginBottom="8dp"
             app:layout_constraintBottom_toBottomOf="parent">
 
             <androidx.appcompat.widget.AppCompatImageView
                 android:id="@+id/backButton"
-                android:layout_width="40dp"
-                android:layout_height="40dp"
-                android:padding="4dp"
-                android:focusable="true"
-                android:clickable="true"
+                android:layout_width="48dp"
+                android:layout_height="48dp"
+                android:layout_marginEnd="8dp"
                 android:background="?attr/selectableItemBackgroundBorderless"
+                android:clickable="true"
                 android:contentDescription="@null"
-                android:layout_marginEnd="32dp"
-                app:tint="?attr/material_theme_de_emphasised_color"
-                app:srcCompat="@drawable/ic_replay_black_24dp"
-                app:layout_constraintEnd_toStartOf="@+id/nextButton"
+                android:focusable="true"
+                android:padding="12dp"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toStartOf="@+id/addDescriptionButton"
                 app:layout_constraintTop_toTopOf="parent"
-                app:layout_constraintBottom_toBottomOf="parent"/>
+                app:srcCompat="@drawable/ic_arrow_back_themed_24dp"
+                app:tint="?attr/colorAccent" />
 
-            <com.google.android.material.floatingactionbutton.FloatingActionButton
-                android:id="@+id/nextButton"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
+            <LinearLayout
+                android:id="@+id/addDescriptionButton"
+                android:layout_width="194dp"
+                android:layout_height="48dp"
                 android:layout_marginTop="8dp"
                 android:layout_marginBottom="8dp"
-                android:contentDescription="@string/view_next_random_article"
-                app:backgroundTint="?attr/paper_color"
-                app:borderWidth="0dp"
+                android:background="@drawable/button_shape_gray"
+                android:backgroundTint="?attr/colorAccent"
+                android:foreground="?attr/selectableItemBackgroundBorderless"
```

Comment:
Looks like it does not have the correct ripple animation and will show over the radius corners.

Reply:
Right! done!